data/ib_buffer_pool

```diff
@@ -1,210 +1,109 @@
-3,3
-0,51
+3,2
```

Comment:
**Do this one last**

![image](https://user-images.githubusercontent.com/35232348/114877162-9c6e6a00-9e31-11eb-8f44-8829a51af090.png)
Add these lines of code to your .gitignore

After you are done editing everything, follow these commands:
`git rm -r --cached data`

`git add .`

`git commit -m "Fixing data and fixing spaces"`

`git push -u origin static_pages`